styles/style.css

```diff
@@ -0,0 +1,699 @@
+  background-image: url(../img/2013-tesla-model-s-cockpit.png);
+  background-position: bottom left;
+  background-repeat: no-repeat;
+}
+
+.top-title {
+  color: #E51937;
+  font-size: 70px;
+  font-weight: 600;
+  line-height: 85px;
+  line-height: 85px;
+  margin: 40px 0 0 40px;
+}
+
+.info__description-advantage {
+  max-width: 423px;
+  color: #fff;
+  background: #E51937;
+  padding: 55px 45px;
+}
+
+.advantage-text {
+  font-size: 30px;
+  line-height: 37px;
+  font-weight: 600;
+  margin: 0;
+}
+
+.advantage-subtext {
+  font-size: 15px;
+  line-height: 18px;
+  font-weight: 500;
+  margin-bottom: 55px;
+}
+
+.image-advantage {
+  margin: 0 0 160px 0;
+}
+
+.advantage_info-btn {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  white-space: nowrap;
+}
+
+.info_btn-text {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 13px;
+  font-weight: 600;
+  font-style: normal;
+}
+
+.info-btn__img {
+  margin-right: 5px;
+}
+
+.btn-info {
+  font-size: 13px;
+  font-weight: bold;
+  font-style: normal;
+  text-transform: uppercase;
+  color: #fff;
+  display: block;
+  padding: 10px 43px; 
+  border: 2px solid #fff;
+  background: #E51937;
+  cursor: pointer;
+  transition-duration: .4s;
+  margin-left: 18px;
+}
+
+.btn-info:hover {
+  background: #fff;
+  color: #E51937;
+}
+
+.safety-info {
+  display: flex;
+}
+
+.rating-safety {
+  width: 440px;
+  color: #fff;
+  background: #E51937;
+  padding: 55px 45px;
+}
+
+.safety-text {
+  font-size: 30px;
+  font-weight: 600;
+  font-style: normal;
+  line-height: 37px;
+  margin: 0;
+}
+
+.safety-subtext {
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 18px;
+  font-style: normal;
+  margin-bottom: 55px;
+  padding-top: 15px;
+}
+
+.rating_title-text {
+  font-size: 30px;
+  font-weight: 600;
+  line-height: 37px;
+  font-style: normal;
+  margin: 0;
+}
+
+.stars {
+  list-style: none;
+  padding: 0;
+  margin: 20px 0 70px 0;
+}
+
+.star-item {
+  font-size: 17px;
+  font-weight: bold;
+  line-height: 21px;
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-bottom: 10px;
+}
+
+.star-item span {
+  font-size: 10px;
+  line-height: 12px;
+  opacity: .6;
+  padding-top: 5px;
+}
+
+.star-group {
+  width: 230px;
+  padding-left: 30px;
+}
+
+.rating_icon {
+  width: 280px;
+  height: 90px;
+  margin-bottom: 50px;
+}
+
+.safety_info-btn {
+  width: 319px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  white-space: nowrap;
+}
+
+.safety_info-text {
+  font-size: 13px;
+  font-weight: 600;
+  font-style: normal;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.safety-info__img {
+  margin-right: 5px;
+}
+
+.btn-safety {
+  font-size: 13px;
+  font-weight: bold;
+  font-style: normal;
+  text-transform: uppercase;
+  color: #fff;
+  display: block;
+  margin-left: 10px;
+  padding: 10px 43px; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)